src/App.tsx

```diff
@@ -1,36 +1,35 @@
-import React, { useState } from 'react';
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 
-import { About } from './components/About/About.tsx';
-import { CartProvider } from './components/CartContent/CartContent.tsx';
-import useTheme from './components/customHooks/useTheme.ts';
-import { Footer } from './components/footer/Footer.tsx';
-import { Header } from './components/header/Header.tsx';
-import { Products } from './components/Products/Products.tsx';
-import { Page } from './pages.ts';
+import { About } from './components/About/About';
+import { CartProvider } from './components/CartContent/CartContent';
+import { ThemeProvider } from './components/CartContent/ThemeContext';
+import useTheme from './components/customHooks/useTheme';
+import LayoutComponent from './components/LayoutComponent/LayoutComponent';
+import PageNotFound from './components/PageNotFound/PageNotFound';
+import ProductPage from './components/ProductPage/ProductPage';
+import { Products } from './components/Products/Products';
 
 import './App.css';
 
 function App() {
-    const [currentPage, setCurrentPage] = useState<Page>(Page.PRODUCTS);
     const { isDarkTheme, handleThemeChange } = useTheme();
 
-    const handlePageChange = (page: Page) => {
-        setCurrentPage(page);
-    };
-
-    const pageComponents = {
-        [Page.ABOUT]: <About />,
-        [Page.PRODUCTS]: <Products isDarkTheme={isDarkTheme} />,
-    } as const;
-
     return (
-        <>
-            <CartProvider>
-                <Header onPageChange={handlePageChange} onThemeChange={handleThemeChange} isDarkMode={isDarkTheme} />
-                {pageComponents[currentPage]}
-                <Footer />
-            </CartProvider>
-        </>
+        <Router basename="/fe-react-2024">
+            <ThemeProvider>
+                <CartProvider>
+                    <Routes>
+                        <Route path="/" element={<LayoutComponent onThemeChange={handleThemeChange} isDarkMode={isDarkTheme} />}>
```

Comment:
u created context for theme, so why u sending `isDarkTheme` as a property to the component, u case invoke context inside, and remove the comment

Reply:
I've been trying to fix this but I keep having something break and stop working, I can't figure out why... Help with this, plz

Reply:
okey, will check it out, in evening

Reply:
Thanks🙏🏼